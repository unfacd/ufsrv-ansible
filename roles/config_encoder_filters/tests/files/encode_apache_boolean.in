content:
  - options:
      - option1: yes
      - option2: no
      - option3: "true"
      - option4: "False"
