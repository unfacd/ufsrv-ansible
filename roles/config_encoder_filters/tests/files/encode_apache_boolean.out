option1 true
option2 false
option3 true
option4 False
